### DISCUSSION TITLE
Simple large durations

### DISCUSSION THREAD

**Discussion by argyleink** [argyleink/open-props#398] [Ideas]
Usage like `var(--1day)` or something 

- 1minute
- 5minutes
- .5hour
- .5day
- 1day
- 2days
- 1week
- etc

Def be a non-main bundle package, but nice to import if needed, and also cool to have in the jit-props set

Or is this a job for custom units?

**Comment by igrep** [comment]
I'm worried if the abbreviations 1m, 1d and 1wk  are so short that they conflict other words. In addition, using abbreviations is inconsistent with the other existing open-props' constants because most of them are not abbreviated. That is one of the things where I like open-props better than the other frameworks such as Tailwind.

**Reply by argyleink** [reply]
ty, excellent comment and suggestion. will update the initial proposal accordingly 👍🏻
